tests/integration/etcdsnapshot/etcdsnapshot_int_test.go

```diff
@@ -111,6 +120,100 @@ var _ = Describe("etcd snapshots", Ordered, func() {
 			}
 		})
 	})
+	When("a new etcd is created with server start flags", func() {
+		It("kills previous server and start up with no problems", func() {
+
+			var err error
+			Expect(testutil.K3sKillServer(server)).To(Succeed())
+			localServerArgs := []string{"--cluster-init",
+				"--etcd-snapshot-name", etcdSnapshotFilePattern,
+				"--etcd-snapshot-dir", populatedTestSnapshotDir,
+				"--etcd-snapshot-retention", etcdSnapshotRetention,
+				"--etcd-snapshot-schedule-cron", `* * * * *`,
+				"--etcd-snapshot-compress"}
+			server, err = testutil.K3sStartServer(localServerArgs)
+			Expect(err).ToNot(HaveOccurred())
+			Eventually(func() error {
+				return testutil.K3sDefaultDeployments()
+			}, "180s", "5s").Should(Succeed())
+
+		})
+		It("saves an etcd snapshot with specified name and it should be no more than 1 compressed file", func() {
+			Eventually(func() error {
```

Comment:
Since what we want is only 1 file to exist over a period of time, it may be best to use gomega `Consistently` testing function ( see the [Gomega docs](https://onsi.github.io/gomega/#consistently) on this). The function should return a single `int` value, which we run the check against.